backend/tests/test1.test.js

```
@@ -0,0 +1,285 @@
+      save: jest.fn(async () => base),
+      toObject: function () {
+        return { ...base };
+      },
+    };
+  };
+  const makeQuery = (doc) => {
+    const wrapped = wrapDoc(doc);
+    const thenableDoc = wrapped
+      ? {
+          ...wrapped,
+          then: (res, rej) => Promise.resolve(wrapped).then(res, rej),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)